app/controllers/games_controller.rb

```diff
@@ -0,0 +1,28 @@
+class GamesController < ApplicationController
+	# before_action :authenticate_user!
+  
+  def show
+  	@game = Game.find(params[:id])
+  end
+
+  def new
+  	@game = Game.new
+  end
+
+  def create
+  	@game = Game.new(game_params)
+  	if @game.save
+  		redirect_to game_path(@game)
+  	else
+  		render :new, :status => :unprocessable_entity 
+  	end
+
+  end
+
+  private
+
+  def game_params
+		params.require(:game).permit(:name)  	
+  end
+
+end
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5273755/11350083/a83d65de-91e4-11e5-862d-4d219349fa37.png)
You can also have sublime reindent any file:

- copy everything
- hit CMD + shift + p
- search for `reindent`

It doesn't *always* works but its close.